doc/content/reference/root-certificates/ca.pem

```diff
@@ -0,0 +1,146 @@
+DST Root CA X3
+==============
```

Comment:
This is not a valid `.pem` file. You should remove those headings and also the extra new line. Here's a [reference](https://github.com/ARMmbed/mbedtls/blob/development/tests/data_files/test-ca_cat21.crt)

If you want to tag which certificate is which, I'd recommend adding an extra `md` file somewhere and listing these in order.

Reply:
I think anything outside the certificate tags is ignored https://stackoverflow.com/questions/19578812/comments-in-a-pem-file

Reply:
That's not standard behaviour. Some clients (like `OpenSSL`) do that but not all.

Reply:
Anything outside the boundaries is indeed ignored: https://tools.ietf.org/html/rfc7468#section-2

Reply:
This is fine: https://tools.ietf.org/html/rfc7468#section-5.2